_inc/client/at-a-glance/stats/index.jsx

```diff
@@ -187,7 +187,7 @@ export class DashStats extends Component {
 			const statsChart = this.statsChart( this.props.activeTab ),
 				chartData = statsChart.chartData,
 				totalViews = statsChart.totalViews,
-				showEmptyStats = totalViews <= 0 && ! this.props.isEmptyStatsCardDismissed && ! this.state.emptyStatsDismissed;
+				showEmptyStats = chartData.length > 0 && totalViews <= 0 && ! this.props.isEmptyStatsCardDismissed && ! this.state.emptyStatsDismissed;
```

Comment:
`.length` is always a positive number, or zero. It's safe to just check for `charData.length`